docs/user-management.md

```diff
@@ -1,5 +1,12 @@
 # User Management
 
+Kubernetes, and thus k0s, does not have any built-in functionality to manage users. Kubernetes relies solely on external sources for user identification and authentication. A client certificate is considered an external source in this case as Kubernetes api-server "just" validates that the certificate is signed by a trusted CA. This means that it is recommended to use e.g. [OIDC configuration](./examples/oidc/oidc-cluster-configuration.md) to configure API server to trust the external OIDC provider issued user tokens.
+
+k0s comes with some helper commands to create kubeconfig with client certificates for users. There are few caveats that one needs to take into consideration when using client certificates:
+
+* Client certificates have long expiration time, they're valid for one year
+* Client certificates cannot be revoked
```

Comment:
NB: That isn't generally true for client certificates, but just applies to the built in capabilities of k0s.